src/lib/snark_work_lib/partitioned_result.ml

```diff
@@ -19,25 +18,18 @@ module Stable = struct
 end]
 
 type t =
-  ( Transaction_witness.t
-  , Ledger_proof.Cached.t
-  , Sub_zkapp_spec.t
+  ( unit
+  , unit
```

Comment:
I wonder, is it useful to have a version of this type with `Ledger_proof.Cached.t`?

Maybe we could just use `Ledger_proof.t`? It looks like this structure is scoped to a particular context, and it doesn't feel like we'll need cached proofs in that context.

We don't need to cache subzkapp proofs, I think. They are shortlived.